tests/OpenApiRouteTest.php

```diff
@@ -38,11 +41,11 @@ public function openApiFileProvider() : array
      */
     public function it_sets_server_to_current_app_url_if_modify_file_is_enabled($openApiFile)
     {
-        config()->set('swagger-ui.file', $openApiFile);
-        config()->set('swagger-ui.modify_file', true);
+        config()->set('swagger-ui.files.0.versions', ['v1' => $openApiFile]);
```

Comment:
when openApiFileProvider uses the "multiple versions" dataset, then this will create an invalid config. I don't think we should use the dataprovider for this